enos/ansible/roles/utils/tasks/test.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Adding flent repository
+  apt_repository:
+    repo: deb http://download.opensuse.org/repositories/home:/tohojo:/flent/xUbuntu_14.04/ /
```

Comment:
We recently switch to Debian jessie. Shall we use `http://download.opensuse.org/repositories/home:/tohojo:/flent/Debian_8.0/`?